### PR TITLE
tmuxPlugins.fingers: init at 1.0.1

### PR DESCRIPTION
**Pull request by ceedubs**
###### Motivation for this change

[tmux-fingers](https://github.com/Morantron/tmux-fingers) provides copy pasting with vimium/vimperator like hints.

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [x] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/misc/tmux-plugins/default.nix [hunk 1]
```diff
@@ -109,6 +109,19 @@ in rec {
     };
   };
 
+  fingers = mkDerivation {
+    pluginName = "fingers";
+    version = "1.0.1";
+    src = fetchFromGitHub {
+      owner = "Morantron";
+      repo = "tmux-fingers";
+      rev = "d722ec7b3af18924f902a77c9e0beda5ca643f04";
```
Comment (drewrisinger) [inline pkgs/misc/tmux-plugins/default.nix:118]:
```suggestion
      rev = version;
```
This will check out the tag corresponding to v1.0.1

**Review by drewrisinger — CHANGES_REQUESTED** [review]
Change checked-out rev

Built locally with ``nixpkgs-review``, didn't test beyond that.

**Comment by drewrisinger** [comment]
These will need squashed down. And to make OfBorg happy, you should change title to something like ``tmuxplugin.tmux-fingers`` (sorry, I don't know the exact prefix, it's however you would build it via ``nix-build -A ...``).
This currently fails to evaluate because you haven't defined the variable ``version`` in the proper scope. You need to make the derivation a recursive set, like in 
https://github.com/NixOS/nixpkgs/blob/7e9f153d6f06ac02f01fe12438d3544214f7895b/pkgs/misc/tmux-plugins/default.nix#L101

**Comment by ceedubs** [comment]
Thank you for the helpful review, @drewrisinger! Hopefully I've resolved all of the issues.

**Comment by drewrisinger** [comment]
Leaving it as ``tmuxPlugs.tmux-fingers`` is fine with me if you'd like. I'm not familiar with the conventions, basically whatever name will make it easier to find for users.

@GrahamcOfBorg build tmuxPlugsin.fingers

**Comment by ceedubs** [comment]
@drewrisinger taking a look through https://github.com/NixOS/nixpkgs/blob/master/pkgs/misc/tmux-plugins/default.nix it looks like the tmux plugins generally don't have a `tmux-` prefix, so I _think_ that in its current form it's matching the convention.

**Review by drewrisinger — APPROVED** [review]

**Review by jonringer — APPROVED** [review]
diff LGTM

```
https://github.com/NixOS/nixpkgs/pull/85529
1 package built:
tmuxPlugins.fingers
```

**Pull request merged**